### ISSUE TITLE
How is the project going?

### ISSUE THREAD

**Issue by TunaYagci** [csm/jycrypto#1]
Hey there sir,

Thank you for starting this project. What's its development state? I hope you can remember, as it's been almost 5 years since the first commit :smile:

**Comment by csm** [comment]
Yeah, sorry for the abandonware, but I haven't fleshed this out at all, and don't really remember what state it was in last I left it.
